doc/setup/installing_linux_python.adoc

```diff
@@ -0,0 +1,43 @@
+= Installing OSI for Python on Linux
+
+**Prerequisites**
+
+* You have installed _pip3_.
+* You have installed _python-setuptools_.
+* You have installed _protobuf_.
+* For local installation, you have installed _virtualenv_.
+
+**Steps**
+
+. Open a terminal.
+. Clone the Open Simulation repository.
++
+[source]
+----
+git clone https://github.com/OpenSimulationInterface/open-simulation-interface.git
+----
++
+. Change into the repository directory.
++
+[source]
+----
+cd open-simulation-interface
+----
++
+. Create a new virtual environment.
++
+[source]
+----
+virtualenv -p python3 venv
+----
++
+. Activate the virtual environment.
++
+[source]
+----
+source venv/bin/activate
+----
++
+. Install Open Simulation Interface.
+Local installation:: `pip3 install .`
+Global installation:: `sudo pip3 install .`
```

Comment:
Please review @pmai

Reply:
Looks good to me, any further tuning of installation guidance can be done when we improve installation/cmake stuff.